README.md

```diff
@@ -33,9 +33,12 @@ Module usage:
 | lifecycle\_days\_to\_infrequent\_storage\_transition | Specifies the number of days after object creation when it will be moved to standard infrequent access storage. | string | `"60"` | no |
 | lifecycle\_expiration\_enabled | Specifies expiration lifecycle rule status. | string | `"false"` | no |
 | lifecycle\_expiration\_object\_prefix | Object key prefix identifying one or more objects to which the lifecycle rule applies. | string | `""` | no |
+| lifecycle\_expiration\_object\_tags | Object tags to filter on for the lifecycle rule. | map | `<map>` | no |
```

Comment:
nitpick - in the description for every tag, would be nice to have it specify for which lifecycle rule it's referring to
e.g.
lifecycle\_expiration\_object\_tags | Object tags to filter on for the _expiration_ lifecycle rule.